examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3237,6 +3237,195 @@ server cluster ActivatedCarbonFilterMonitoring = 114 {
   command ResetCondition(): DefaultSuccess = 0;
 }
 
+/** This cluster provides an interface to the functionality of Smart Energy Demand Response and Load Control. */
+provisional server cluster DemandResponseLoadControl = 150 {
+  enum CriticalityLevelEnum : enum8 {
+    kUnknown = 0;
+    kGreen = 1;
+    kLevel1 = 2;
+    kLevel2 = 3;
+    kLevel3 = 4;
+    kLevel4 = 5;
+    kLevel5 = 6;
+    kEmergency = 7;
+    kPlannedOutage = 8;
+    kServiceDisconnect = 9;
+  }
+
+  enum HeatingSourceEnum : enum8 {
+    kAny = 0;
+    kElectric = 1;
+    kNonElectric = 2;
+  }
+
+  enum LoadControlEventChangeSourceEnum : enum8 {
+    kAutomatic = 0;
+    kUserAction = 1;
+  }
+
+  enum LoadControlEventStatusEnum : enum8 {
+    kUnknown = 0;
```

Comment:
specs does not mention 0 on its enum list ; it should be added

Reply:
i bet following other enum this would be named `invalid`

Reply:
Please see PR description: this is targeting an outstanding spec PR, not the already-merged spec.